src/command.rs

```diff
@@ -252,6 +252,8 @@ pub async fn install() -> miette::Result<()> {
     let credentials = Credentials::load().await?;
 
     store.clear().await?;
+    PackageStore::create(store.proto_path()).await?;
```

Comment:
This should be done when calling `open` to prevent the issue across all commands